app/uk/gov/hmrc/fileupload/controllers/EnvelopeController.scala

```diff
@@ -60,7 +60,7 @@ object EnvelopeController extends BaseController {
   def show(id: String) = Action.async{
 
 	  def findEnvelopeFor = (id: String) =>  (envelopeService ? GetEnvelope(id)).mapTo[Envelope]
-	  def toJson = (e: Envelope) => marshaller ? Marshall(e)
+	  def toJson = (e: Envelope) => marshaller ? e
```

Comment:
I have added a specific method since we are specific in the 'generic' one. Maybe we should delete that one.